web/styles/app_components.css

```diff
@@ -471,16 +466,6 @@ input.settings_text_input {
     margin: 15px 0 -5px 23px;
 }
 
-.topic-list-filter {
-    text-overflow: ellipsis;
-    overflow: hidden;
-    white-space: nowrap;
-
-    & input {
-        padding-right: 20px;
-    }
-}
```

Comment:
Why is topic list code changed in the settings commit? If it's because you're editing `clear_search_button`, I would suggest doing something so you can keep the commits more separated. One option would be temporarily renaming `clear_search_button` to `old_clear_search_button` everywhere, then start a new `clear_search_button` with just the styles you're keeping and apply that to each close button as you migrate each of them, and then delete `old_clear_search_button` at the end.